internal/core/services/autoscaler/autoscaler_test.go

```diff
@@ -227,6 +227,25 @@ func TestCanDownscale(t *testing.T) {
 
 			assert.True(t, allow)
 		})
+
+		t.Run("When the ready target is low", func(t *testing.T) {
```

Comment:
I think the name of the test could describe what it should happen when the ready target is low